lightweight-term-count-update.php

```diff
@@ -57,7 +57,7 @@ private function __construct() {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
-			self::$instance = new static;
+			self::$instance = new static();
```

Comment:
Does this relate to solving the problem this PR addresses?

Reply:
This was also throwing a PHPCS error causing the build job to fail: https://travis-ci.org/Automattic/lightweight-term-count-update/jobs/376770856#L644